0014-http-ilp/sequence.mmd

```diff
@@ -0,0 +1,29 @@
+sequenceDiagram
+participant Payer
+participant Interledger
+participant PayeeWallet
+participant Payee
+activate Payer
+Payer->>+Payee: Attempt Request
+Payee-->>-Payer: 402 Payment Required
```

Comment:
If this is describing the `OPTIONS` flow, it should begin with an OPTIONS request that ends with a 204, rather than a regular request that ends with a 402.